src/lib/statistics-service/recorder.ts

```diff
@@ -38,16 +38,11 @@ function skipSendEventToRecorder(event: RecorderEvent): boolean {
   return event.benefits ? event.benefits.length === 0 : true
 }
 
-function identifyEvent(matomo: Matomo | undefined): string {
-  return matomo !== undefined
-    ? matomo.getVisitorId()
-    : `uid_${Math.random().toString(12).slice(2)}`
+function identifyEvent(): string {
+  return tracker.getVisitorId() || `uid_${Math.random().toString(12).slice(2)}`
```

Comment:
Soucis sur le type de retour, getVisitorId de Matomo est async :
```suggestion
async function identifyEvent(): Promise<string> {
  const visitorId = await tracker.getVisitorId()
  return visitorId || `uid_${Math.random().toString(12).slice(2)}`
```

Reply:
Ah tu as trouvé ça où ? j'avais fait des test en console + check la doc et ça avait l'air d'être bon ?

Reply:
Mince j'ai tout mis en async mais je crois pas que ce soit le cas en vrai, je vais re-creuser

Reply:
Ah my bad c'est pas async en fait